doc/external-api/README.md

```diff
@@ -82,6 +82,15 @@ $ ./main
 Nix version: 2.17
 ```
 
+You might need to add the directory containing `nix-expr-c.pc` to the
+`PKG_CONFIG_PATH` environment variable. For example after [building
+Nix](https://github.com/NixOS/nix/blob/master/doc/manual/src/contributing/hacking.md#building-nix-with-flakes)
+by executing the following.
+
+```ShellSession
+$ export PKG_CONFIG_PATH=$(realpath outputs/out/lib/pkgconfig):$PKG_CONFIG_PATH
+```
+
```

Comment:
I think ideally we'd use `nix develop --redirect` to hook into the things Nixpkgs `setup.sh` does, but to be fair, so far I've preferred the robustness of just doing a full build on a fast machine.

Reply:
good point. but this looks tricky and i do not think i can do this at the moment. so i will split this pull request so the other parts can already be merged.